webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/angular/nodeProperties.js

```diff
@@ -163,7 +163,6 @@ nodePropertiesApp.controller('nodePropertiesCtrl', function ($scope, $http, DTOp
       if(!$scope.alreadyUsed){
         $http.post($scope.urlAPI, data).then(function successCallback(response) {
           $scope.errorSaving = false;
-          $scope.properties.push(propertyToSave);
```

Comment:
The problem cames from the fact that the list of properties is updated twice when a property is added/modified.

This line is not necessary since we call `$scope.fetchProperties()` in a second step, which will fetch the list of all properties (plus the one we just added/modified) and update **$scope.properties** with the result.

Reply:
why is not always failing ?